sirepo/package_data/static/js/controls.js

```diff
@@ -219,6 +244,33 @@ SIREPO.app.controller('ControlsController', function(appState, controlsService,
         return res;
     }
 
+    function getInitialMonitorPositions() {
+        const o = new MutationObserver(function (mutations) {
```

Comment:
@moellep I have to imagine there is a better way to do this. What I want to do is what until the readout-table is rendered in the DOM before trying to apply the classes which would be a no-op. We have things like panelState.waitForUI but nothing besides this seemed to work outside of a single directive. I also tried using rootScope.broadcast and a scope.on here but the broadcast happens multiple times and I de-registering the listener seemed to not always work.

Reply:
I changed it to a boolean: controlsService.isReadoutTableActive and then refer to that value whenever the readout table is converted to html.